crates/core/src/db/messages/transaction.rs

```diff
@@ -23,6 +23,7 @@ pub struct Transaction {
     pub writes: Vec<Write>,
 }
 
+#[allow(clippy::items_after_test_module)]
```

Comment:
:facepalm:

Reply:
Opened rust-lang/rust-clippy#12247 bc of this